api/py/ai/chronon/repo/__init__.py

```diff
@@ -12,9 +11,17 @@
 #     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
+from enum import Enum
 
-JOIN_FOLDER_NAME = 'joins'
-GROUP_BY_FOLDER_NAME = 'group_bys'
-STAGING_QUERY_FOLDER_NAME = 'staging_queries'
+JOIN_FOLDER_NAME = "joins"
+GROUP_BY_FOLDER_NAME = "group_bys"
+STAGING_QUERY_FOLDER_NAME = "staging_queries"
 # TODO - make team part of thrift API?
-TEAMS_FILE_PATH = 'teams.json'
+TEAMS_FILE_PATH = "teams.json"
+
+
+class FeatureDisplayKeys(Enum):
+    SOURCE_KEYS = "source_keys"
+    INTERNAL_COLUMNS = "internal_columns"
+    EXTERNAL_COLUMNS = "external_columns"
+    DERIVED_COLUMNS = "derived_columns"
```

Comment:
How come we add these here, and not in `utils`?

Reply:
hmm, yeah. I could put this in utils. Moved.